packages/connect/src/device/Device.ts

```diff
@@ -330,6 +336,27 @@ export class Device extends TypedEmitter<DeviceEvents> {
         return this.releasePromise;
     }
 
+    private async setupThp() {
+        _log.info('Setup THP device');
+        this._protocol = protocolV2;
+
+        if (
+            this.transport.name === 'BridgeTransport' &&
+            !versionUtils.isNewerOrEqual(this.transport.version, '3.1.0')
```

Comment:
in general, there is no standalone bridge version out there, so it should be enought to check just for startsWith('3'). Also bridge version, in the wild, is reported as <bridge-version>.<suite-version>. so this check probably wouldn't work.